jcommune-view/jcommune-web-view/src/main/webapp/resources/javascript/app/userSearch.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2011  JTalks.org Team
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+var UserSearch = {};
+
+UserSearch.groupSelected = function(userID, groupID) {
+    $.ajax({
+        url: $root + "/user/" + userID + "/group/" + groupID,
+        type: 'POST',
+        success: function(result) {
+            console.log("group " + groupID + " added for user " + userID);
+        }
+    });
+};
+
+UserSearch.groupDeleted = function(userID, groupID) {
+    $.ajax({
+        url: $root + "/user/" + userID + "/group/" + groupID,
+        type: 'DELETE',
+        success: function(result) {
+            console.log("group " + groupID + " deleted for user " + userID);
+        }
+    });
+};
+
+UserSearch.toggleUserGroups = function(userID) {
+    $('#user-groups-table-' + userID).toggle(0, function() {
+        if($(this).is(":visible")) {
+            window.UserSearch.showUserGroups(userID);
```

Comment:
`window` part is probably not needed here - use can simply use UserSearch. Or you can do this in object oriented way with the classes.